python/tskit/trees.py

```diff
@@ -357,6 +357,19 @@ def __eq__(self, other):
             and self.metadata == other.metadata
         )
 
+    def alleles(self):
```

Comment:
Adding a type hint here would be nice seeing we're returning a set - it gets copied through to the docs.